content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -289,6 +293,8 @@ using (MappedDiagnosticsLogicalContext.SetScoped("dd.span_id", CorrelationIdenti
 {{% /tab %}}
 {{% tab "Microsoft.Extensions.Logging" %}}
 
+*Note:* If you are using Serilog as an ILogger provider, it is preferrably to use this `ILogger.BeginScope()` method as opposed to the Serilog static properties. Passing an `ILogger.BeginScope()` an `IDictionary<string,object>` as the same effect as `LogContext.PushProperty()`. See [this blog by the author of Serilog][13].
```

Comment:
@zippy1981 from your experience, do you think this note is required or do you think we can remove this additional text? The other edits you've provided will redirect ILogger/Serilog users here, and the steps below only indicate how to use `ILogger.BeginScope` since it is our recommended path. I'm thinking it may be unnecessary text and that removing it could remove some confusion, but I wonder what you think?

Reply:
The key thing for me is pointing people towards that blog by the author of Serilog on how Beginscope works. I do agree this note is probably redundant, so where do you suggest I direct people towards that blog.

Reply:
@zippy1981 @zacharycmontoya 
What about adding a line like this at the end of this section:

You can read more about using BeginScope in this blog post, [The semantics of ILogger.BeginScope()][13].